package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^14.4.3",
     "axios": "^1.4.0",
     "bootstrap": "^5.2.3",
+    "bootstrap-icons": "^1.10.5",
```

Comment:
Bootstrap-icon is not required. In pakage.json, there is react icon library already available. Please use it for icons. To learn it [use this link](https://react-icons.github.io/react-icons/).
(PN: please skip the installation part since it is already in package.json as a dependency"